timeseries/src/autogluon/timeseries/evaluator.py

```diff
@@ -228,7 +256,15 @@ def score_with_saved_past_metrics(
         it doesn't require splitting the test data into past/future portions each time (e.g., when fitting ensembles).
         """
         assert (predictions.num_timesteps_per_item() == self.prediction_length).all()
-        assert self._past_naive_error is not None, "Call save_past_metrics before score_with_saved_past_metrics"
+        if self.eval_metric == "MASE":
```

Comment:
Nit: a shorter way to check this would be
```python
if self.eval_metric == "MASE" and self._past_abs_seasonal_error is None:
    raise AssertionError(...)
```

Reply:
Done!